src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -255,7 +257,15 @@ public StateFlow build(FlowState.Main initialState) {
             .onlyIf((respondent1NotRepresented.and(respondent2NotRepresented))
                         .or(respondent1NotRepresented.and(respondent2OrgNotRegistered.negate()))
                         .or(respondent1OrgNotRegistered.negate().and(respondent2NotRepresented))
-                        .and(not(specClaim)))
+                        .and(not(specClaim))
+                        .and(not(certificateOfServiceEnabled)))
+            //Added this condition temporarly until release of certif of Service.
```

Comment:
sorry didnt quite follow this comment ?

Reply:
As we have introduced new state "PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT_COS" only for COS thats why this state will only transited if CoS is not enabled. New state intorduced for not changing current functionality in production and allowing the user event "Notify claim" only when COS is enabled.